### DISCUSSION TITLE
Let's start the discussion 🥳

### DISCUSSION THREAD

**Discussion by foyzulkarim** [foyzulkarim/mernboilerplate-antd#23] [Announcements]
যারা মার্ন বয়লারপ্লেটে কাজ করতে গিয়ে আটকে যাচ্ছেন, বা এমনিতেই কিছু আলোচনা করতে চাচ্ছেন তারা কিন্তু চাইলে গিটহাব এর ডিসকাশন ফিচার টা ব্যাবহার করতে পারেন। এমনকি এটাও জিজ্ঞেস করতে পারেন কিভাবে কি শুরু করবেন। ইন শা আল্লাহ কাজ করতে করতে তাহলে নিজেকে আর একা মনে হবে না। 
আমার ধারণা, যেহেতু এখন অনেক ভাইয়েরাই এটা নিয়ে কাজ করার চেষ্টা করতেছেন, আপনাদের মনে অনেক রকমের প্রশ্ন বা কেন কি কিভাবে হচ্ছে সেগুলো নিয়ে জিজ্ঞাসা থাকতে পারে। আপনারা আলোচনা শুরু করলে আশা করতেছি আমার সাথে অন্য ভাইয়েরাও আলোচনায় যোগ দিবেন ইন শা আল্লাহ। 
ডিসকাশন এর ফলে আপনার Writing Skill and  Communication skill  দুইটাই বাড়বে ইন শা আল্লাহ। বাংলাতেও প্রশ্ন করতে পারেন চাইলে, ভাষার মাসে বাংলাকে আমরা ছড়িয়ে দেই গিটহাব এ. 
বিজ্ঞাপন এর মতো বলতে হয়: তাহলে দেরী কেন? এখনই ডিসকাশন শুরু করুন। 🥳

**Comment by findridoy** [comment]
Love this. As a Bangladeshi I miss this type of open-source Bangladeshi product. I will definitely be part of this as soon as possible.
